src/client/rooms/room-page.js

```diff
@@ -30,19 +33,39 @@ export class RoomPage extends Component {
     }
 
     render() {
+        let round;
+        switch (this.props.nextRoundType) {
```

Comment:
round type should be mapped to some kind of constant, 'L' and 'N' doesn't mean anything _really_